build.gradle

```diff
@@ -31,15 +31,20 @@ targetCompatibility = 1.6
 repositories {
   mavenCentral()
   mavenLocal()
+  google()
+  jcenter()
+  maven { url 'https://oss.sonatype.org/content/repositories/snapshots' } // for spoon snapshot
 }
 
 dependencies {
   compile gradleApi()
   compile localGroovy()
 
-  compile 'com.squareup.spoon:spoon-runner:1.7.1'
+  compile 'com.squareup.spoon:spoon-runner:2.0.0-SNAPSHOT'
 
-  compile 'com.android.tools.build:gradle:1.5.0'
+  compile('com.android.tools.build:gradle:3.0.0') {
```

Comment:
Could you update it to 3.0.1 ?

Reply:
Try using https://github.com/jaredsburrows/gradle-spoon-plugin.

Reply:
Actually I am trying :)